middle_end/flambda/inlining/inlining_decision.ml

```diff
@@ -370,9 +366,14 @@ let might_inline dacc ~apply ~function_decl ~simplify_expr ~return_arity
   let code_id = I.code_id function_decl in
   let code = DE.find_code denv code_id in
   let cost_metrics = Code.cost_metrics code in
+  let args =
+    Apply.inlining_arguments apply
+    |> Inlining_arguments.meet (DA.denv dacc |> DE.inlining_arguments)
+  in
   let size = Cost_metrics.size cost_metrics in
-  let round = DE.round denv in
```

Comment:
I wonder if we should actually remove `DE.round` at present, given that we're only doing one round.

Reply:
We could. In that case I'll do it in a separate PR where I'll be discarding [round] completely from the simplifier.